assets/js/components/common/DashboardLayout.jsx

```diff
@@ -171,7 +179,7 @@ class DashboardLayout extends Component {
                       </a>
                     ))}
                     <div style={{ marginLeft: "auto" }}>
-                      © 2021 Helium Systems Inc.
+                      © 2022 Helium Systems Inc.
```

Comment:
If incrementing the date, then should it be Nova?

Reply:
Will defer to @jdgemm

Reply:
Yeah, I don't really know, but figured Helium Systems Inc. wasn't it

Reply:
per Dal, keeping as is